<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1078</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ecuación:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnaceptar">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>560</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
  width: 200px;
  padding: 8px;
  color: black;
  border: 2px solid black;
  text-align: center;
  outline: none;
  text-decoration: none;
}
QPushButton:hover{
 background-color: #fff;
  color: #000;

}</string>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="valorH">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="valorY">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>120</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="mitabla">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>991</width>
     <height>311</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>130</number>
   </attribute>
   <column>
    <property name="text">
     <string>xi</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>yi</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>k1</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>k2</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>k3</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>k4</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>yi+1</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>70</y>
     <width>191</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Valor final x:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="valorA1">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>70</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>120</y>
     <width>191</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Valor inicial y:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>261</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Tamaño de paso h:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ecuacion">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>611</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>70</y>
     <width>191</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Valor inicial x:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="valorA2">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>70</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
